.github/workflows/binary-compatibility-checks.yml

```diff
@@ -43,11 +38,11 @@ jobs:
         run: cp .jvmopts-ci .jvmopts
 
       - name: Compile code
-        run: sbt "+~ ${{ matrix.scalaVersion }} Test/compile"
```

Comment:
There is no reason to also compile `Test` scope, the point of this github check is to validate binary compatibility issues and this is never done against test code.